lua/sqls/commands.lua

```diff
@@ -7,6 +7,9 @@ local M = {}
 
 local function make_show_results_handler(mods)
     return function(err, result, _, _)
+        if not result then
```

Comment:
I think checking the value of `result` before `err` silences errors, could you move these checks below `if err`? Otherwise LGTM, thank you for the PR!

Reply:
You are right, I'll change it.

Reply:
Thanks again!